docs/Project.toml

```
@@ -2,4 +2,4 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 [compat]
-Documenter = "~0.20"
+Documenter = "~0.23"
```

```suggestion
Documenter = "0.23"
```
(See https://github.com/JuliaDocs/Documenter.jl/pull/1105)